### PR TITLE
Inference for PROs update

### PR DESCRIPTION
**Pull request by pcuenca**

### CODE REVIEW

File: inference-pro.md [hunk 1]
```diff
@@ -38,11 +40,18 @@ In addition to thousands of public models available in the Hub, PRO users get fr
...
+| Falcon 180B Chat    | [180B](https://huggingface.co/tiiuae/falcon-180B-chat)                                                                                                                                     | 4k tokens      | Conversational model                  |
+| OpenChat 3.5 | [7B](https://huggingface.co/openchat/openchat-3.5-0106)                                                                                                                                           | 8k tokens      | Great performance for a 7B model      |
 | Stable Diffusion XL | [3B UNet](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0)                                                                                                                 | -              | Generate images                       |
+| Bark | [0.9B](https://huggingface.co/suno/bark)                                                                                                                                                                  | -              | Text to audio generation              |
```
Comment (pcuenca) [inline inference-pro.md:54]: I added all the models with Pro permissions as well as the ones deployed for HuggingChat.

File: inference-pro.md [hunk 2]
```diff
@@ -205,6 +294,37 @@ image = sdxl.text_to_image(
...
+    stream=True,
+    max_tokens=500
+)
+
+# iterate and print stream
+for message in chat_completion:
+    print(message.choices[0].delta.content, end="")
+```
+
+TODO: this doesn't work for me yet. Do we need to redeploy?
```
Comment (pcuenca) [inline inference-pro.md:324]: Doesn't work for me yet. @philschmid is there anything I'm missing, do we need to redeploy? Is it available for any of the other models? (tried to test Code Llama 70B in this example)

Comment (philschmid) [reply inference-pro.md:324]: yes we need to update the servin gcontainer pinging here @drbh @Narsil

Comment (julien-c) [reply inference-pro.md:324]: would be very cool to support this soon so we can amplify it for Inference API!

**Review by julien-c — APPROVED** [review]
looks great! let's amplify our Messages API support 🔥

File: inference-pro.md [hunk 3]
```diff
@@ -38,11 +40,18 @@ In addition to thousands of public models available in the Hub, PRO users get fr
...
 | Code Llama Base     | [7B](https://huggingface.co/codellama/CodeLlama-7b-hf) and [13B](https://huggingface.co/codellama/CodeLlama-13b-hf)                                                                        | 4k tokens      | Autocomplete and infill code          |
 | Code Llama Instruct | [34B](https://huggingface.co/codellama/CodeLlama-34b-Instruct-hf)                                                                                                                          | 16k tokens     | Conversational code assistant         |
+| **New** Code Llama 70B Instruct | [70B](https://huggingface.co/codellama/CodeLlama-34b-Instruct-hf)                                                                                                              | 4k tokens      | Conversational code assistant         |
+| Falcon 180B Chat    | [180B](https://huggingface.co/tiiuae/falcon-180B-chat)                                                                                                                                     | 4k tokens      | Conversational model                  |
```
Comment (osanseviero) [inline inference-pro.md:51]: This might have been removed, worth checking

File: inference-pro.md (hunk 1 above)
Comment (osanseviero) [reply inference-pro.md:54]: Ah nice addition!

**Review by osanseviero — APPROVED** [review]
Very nice! :fire:

**Comment by gary149** [comment]
> 
@osanseviero @julien-c should we also include non-pro models? to have an overview we can point users to until we have something different?

I think it's quite nice to keep it simple and not add too many models here. Maybe a link to a different page?

**Comment by julien-c** [comment]
@philschmid let's aim to update the model pages' inference snippets soon IMO

**Review by gary149 — APPROVED** [review]

File: inference-pro.md (hunk 3 above)
Comment (pcuenca) [reply inference-pro.md:51]: You are right, Falcon 180B seems to no longer be available, will remove. Thanks!

**Comment by Narsil** [comment]
https://github.com/huggingface/api-inference/pull/1775

**Comment by pcuenca** [comment]
The Messages API is working now for several models, including Code Llama 70B Instruct (thanks @Narsil!). Merging now so we can amplify comms.

**Pull request merged**